src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        int indexFigure = new Random().nextInt(AllFigure.values().length);
+        ColorSupplier colorSupplier = new ColorSupplier();
+        switch (AllFigure.values()[indexFigure]) {
+            case ISOSCELES_TRAPEZOID:
+                int firstLeg = new Random().nextInt(100) + 1;
+                int secondLeg = new Random().nextInt(100) + 1;
+                int height = new Random().nextInt(100) + 1;
```

Comment:
let's create Random outside the switch/case and use this object to get random values